substrate/frame/assets/src/impl_fungibles.rs

```diff
@@ -164,7 +164,22 @@ impl<T: Config<I>, I: 'static> fungibles::Unbalanced<T::AccountId> for Pallet<T,
 		Ok(amount)
 	}
 
-	// TODO: #13196 implement deactivate/reactivate once we have inactive balance tracking.
+	fn deactivate(asset: Self::AssetId, amount: Self::Balance) {
+		Asset::<T, I>::mutate(&asset, |maybe_asset| match maybe_asset {
+			Some(ref mut asset) => {
+				// Inactive amount can't exceed supply
+				asset.inactive = asset.inactive.saturating_add(amount).max(asset.supply);
```

Comment:
Seems this be a defensive check if the runtime is trying to set the inactive amt > total supply, probably indicates a bug?